src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -330,17 +330,27 @@ private void ChooseActions(in Entity entity, ref MicrobeAI ai, ref CompoundAbsor
             control.SetMucocystState(ref organelles, ref compoundStorage, entity, false);
         }
 
+        float atpLevel = compounds.GetCompoundAmount(Compound.ATP);
+
         // If this microbe is out of ATP, pick an amount of time to rest
-        if (compounds.GetCompoundAmount(Compound.ATP) < 1.0f)
+        if (atpLevel < 1.0f)
         {
             // Keep the maximum at 95% full, as there is flickering when near full
             ai.ATPThreshold = 0.95f * speciesFocus / Constants.MAX_SPECIES_FOCUS;
         }
 
+        // Even if we are out of ATP and there is microbe nearby, engulf them.
+        // make sure engulfing doesn't kill the cell. The cell won't engulf if it would kill it
```

Comment:
I think this comment line is pretty unclear and either should be removed or reworded.